datawave-micrometer/pom.xml

```
@@ -12,7 +12,7 @@
     <artifactId>datawave-micrometer</artifactId>
 
     <properties>
-        <maven.compiler.source>17</maven.compiler.source>
+        <maven.compiler.source>11</maven.compiler.source>
         <maven.compiler.target>17</maven.compiler.target>
```

You should probably do:

```suggestion
        <maven.compiler.release>11</maven.compiler.release>
        <maven.compiler.source>${maven.compiler.release}</maven.compiler.source>
        <maven.compiler.target>${maven.compiler.release}</maven.compiler.target>
```